types/yeoman-test/tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-    "extends": "dtslint/dt.json"
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-outside-dependencies": false
+    }
```

Comment:
You might want to have a little more context on this matter:
`npm test yeoman-test` is not working because a few types (such as `yeoman-environment`) contain a `package.json` whose dependencies need to be installed beforehand.

Unlike `npm test`, `npm run test-all` installs all necessary types containing `package.json`-files for you in order to perform the tests properly.